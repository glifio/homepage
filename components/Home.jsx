import React from 'react'
import {
  AppTile,
  AppTilesWrapper,
  PrimaryBox,
  Page,
  Spacer,
  OneColumnLargeText
} from '@glif/react-components'
import styled from 'styled-components'

const FilecoinLink = styled.a.attrs(() => ({
  href: 'https://filecoin.io/',
  target: '_blank',
  rel: 'noopenner noreferrer'
}))`
  &:hover {
    color: var(--white);
  }
`

export default function AppsHome() {
  return (
    <Page
      appHeaderLinks={[
        {
          title: 'Blog',
          url: 'https://blog.glif.io'
        },
        {
          title: 'Code',
          url: 'https://github.com/glifio'
        },
        {
          title: 'Discord',
          url: 'https://discord.gg/B9ju5Eu4Rq'
        }
      ]}
    >
      <PrimaryBox>
        <h2>
          Glifs are interoperable apps and tools for{' '}
          <FilecoinLink>Filecoin</FilecoinLink>.
        </h2>
      </PrimaryBox>
      <AppTilesWrapper>
        <AppTile
          title='Wallet'
          description='A lightweight interface for sending and receiving Filecoin.'
          href={process.env.NEXT_PUBLIC_WALLET_URL}
          imgSrc='/bg-sender.jpg'
          imgSrcHover='/bg-sender-hover.jpg'
          small
        />
        <AppTile
          title='Safe'
          oldTileName='Vault'
          description='A Filecoin multisig.'
          href={process.env.NEXT_PUBLIC_SAFE_URL}
          imgSrc='/bg-safe.jpg'
          imgSrcHover='/bg-safe-hover.jpg'
          small
        />
        <AppTile
          title='Explorer'
          beta
          description='A Filecoin network explorer.'
          href={process.env.NEXT_PUBLIC_EXPLORER_URL}
          imgSrc='/bg-explorer.jpg'
          imgSrcHover='/bg-explorer-hover.jpg'
          small
        />
        <AppTile
          title='Verify'
          description='A Filecoin notary service.'
          href={process.env.NEXT_PUBLIC_VERIFIER_URL}
          imgSrc='/bg-verifier.jpg'
          imgSrcHover='/bg-verifier-hover.jpg'
          small
        />
        <AppTile
          title='Nodes'
          description='Hosted Filecoin Nodes.'
          href={process.env.NEXT_PUBLIC_NODE_URL}
          imgSrc='/bg-node.jpg'
          imgSrcHover='/bg-node-hover.jpg'
          small
        />
      </AppTilesWrapper>

      <Spacer />

      <OneColumnLargeText>
        <p>
          Instead of the fortress, there are numerous fortresses now, and
          various people have keys.
        </p>
        <p>
          The treasures are no longer stored in the secluded cellars and tombs
          of the few, but guarded by the many. A new language has evolved, used
          to keep the treasures safe and allow those who watch over them to
          communicate, directly and discretely. The hieroglyphic system consists
          only of geometric shapes, as an homage to the interconnected data
          centers with their cages, racks and cabinets.
        </p>
      </OneColumnLargeText>
    </Page>
  )
}
