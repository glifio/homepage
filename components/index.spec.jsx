import { render, screen } from '@testing-library/react'

import Home from './Home'

describe('Apps home page', () => {
  test('it renders correctly', () => {
    const { container } = render(<Home />)

    expect(screen.getAllByText('Wallet')[0]).toBeInTheDocument()
    expect(screen.getAllByText('Safe')[0]).toBeInTheDocument()
    expect(screen.getAllByText('Explorer')[0]).toBeInTheDocument()
    expect(screen.getAllByText('Verify')[0]).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
