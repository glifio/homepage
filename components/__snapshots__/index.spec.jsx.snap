// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apps home page it renders correctly 1`] = `
.c6 {
  width: auto;
  height: 35px;
}

.c19 {
  width: auto;
  height: 35px;
  margin-bottom: 2em;
}

.c10 {
  position: relative;
  font-size: 28px;
  line-height: 1.3em;
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c10 .hover {
  opacity: 0;
  -webkit-transform: scale(1.4);
  -ms-transform: scale(1.4);
  transform: scale(1.4);
}

.c13 {
  height: 100%;
  width: 100%;
  -o-object-fit: contain;
  object-fit: contain;
  position: absolute;
  top: 0;
}

.c12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  position: relative;
  z-index: 2;
  font-size: 26px;
  line-height: 1;
}

.c11 {
  color: #fff;
  background: #000;
  border-radius: 10px;
  overflow: hidden;
}

.c14 {
  z-index: 1;
  font-size: 26px;
  line-height: 1.3;
}

.c14:hover {
  opacity: 1;
}

.c9 {
  gap: 10px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  gap: 10px;
}

.c2 {
  position: -webkit-sticky;
  position: sticky;
  top: 0;
  z-index: 100;
  padding: 10px 0;
  margin: -10px 0;
  background-color: var(--white-broken);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-column-gap: 20px;
  column-gap: 20px;
  row-gap: 10px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  gap: 20px;
}

.c7 {
  gap: 10px;
}

.c4 {
  color: black;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c4:hover {
  color: black;
}

.c8 {
  padding: 0.5em 0.75em;
  border: 1px solid black;
  border-radius: 2em;
}

.c8:hover,
.c8.active {
  color: var(--purple-medium);
  border-color: var(--purple-medium);
}

.c8:active {
  color: white;
  background: var(--purple-medium);
}

.c21 {
  list-style-type: none;
  margin: 0;
  padding-left: 0;
}

.c22 {
  list-style-type: none;
  margin: 0;
  padding-left: 0;
}

.c23 {
  -webkit-text-decoration: underline;
  text-decoration: underline;
  color: inherit;
}

.c18 {
  background: #272727;
  border-radius: 8px;
  padding: 30px;
  color: white;
  position: relative;
}

.c25 {
  text-transform: uppercase;
  display: block;
}

.c15 {
  position: relative;
  padding: 3rem 1.5rem;
}

.c15 > *:first-child {
  margin-top: 0;
}

.c15 > *:last-child {
  margin-bottom: 0;
}

.c17 {
  font-size: 28px;
}

.c17 p {
  max-width: 670px;
  margin: 2em auto;
}

.c17.primary {
  color: white;
  background-color: var(--purple-medium);
  border-radius: 8px;
}

.c17.primary a:hover {
  color: white;
}

.c16 {
  box-sizing: content-box;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 1920px;
  margin: 0 auto;
  padding: 10px;
  gap: 10px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: calc(100vh - 2 * 10px);
  gap: 10px;
}

.c1 > * {
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
}

.c1 > *:last-child {
  -webkit-flex: 1 0 auto;
  -ms-flex: 1 0 auto;
  flex: 1 0 auto;
}

@media (min-width:480px) {
  .c6 {
    height: 45px;
  }
}

@media (min-width:868px) {
  .c6 {
    height: 55px;
  }
}

@media (min-width:480px) {
  .c19 {
    height: 45px;
  }
}

@media (min-width:868px) {
  .c19 {
    height: 55px;
    position: absolute;
    left: 30px;
    top: 30px;
  }
}

@media (min-width:868px) {
  .c10 {
    width: 100%;
    overflow: hidden;
    padding-bottom: 144%;
  }
}

@media (max-width:868px) {
  .c10 {
    width: 45vw;
    -webkit-scroll-snap-align: start;
    -moz-scroll-snap-align: start;
    -ms-scroll-snap-align: start;
    scroll-snap-align: start;
    -webkit-flex-shrink: 0;
    -ms-flex-negative: 0;
    flex-shrink: 0;
  }
}

@media (max-width:480px) {
  .c10 {
    width: 70vw;
    -webkit-scroll-snap-align: start;
    -moz-scroll-snap-align: start;
    -ms-scroll-snap-align: start;
    scroll-snap-align: start;
    -webkit-flex-shrink: 0;
    -ms-flex-negative: 0;
    flex-shrink: 0;
  }
}

@media (hover:hover) {
  .c10:hover .hover {
    opacity: 1;
  }

  .c10:hover .default {
    opacity: 0;
  }
}

@media (hover:hover) {

}

@media (max-width:480px) {
  .c12 {
    padding: 8px;
  }
}

@media (min-width:480px) {
  .c12 {
    padding: 20px;
  }
}

@media (max-width:868px) {
  .c11 {
    height: 60vw;
    position: relative;
  }
}

@media (max-width:480px) {
  .c11 {
    height: 90vw;
    position: relative;
  }
}

@media (min-width:868px) {
  .c11 {
    position: absolute;
    width: 100%;
    height: 100%;
  }
}

@media (min-width:480px) and (hover:hover) {
  .c14 {
    opacity: 0;
    -webkit-transition: opacity 0.5s;
    transition: opacity 0.5s;
  }
}

@media (max-width:868px) {
  .c14 {
    width: 100%;
    white-space: normal;
    font-size: 13px;
    padding: 15px 5px 0 8px;
  }
}

@media (min-width:868px) {
  .c14 {
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    color: #fff;
    padding: 20px;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-box-pack: end;
    -webkit-justify-content: flex-end;
    -ms-flex-pack: end;
    justify-content: flex-end;
    border-radius: 10px;
    opacity: 0;
  }
}

@media (max-width:868px) {
  .c9 {
    margin-top: calc(20px * 2);
    margin-bottom: calc(20px * 2);
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    overflow-x: scroll;
    -webkit-scrollbar-width: none;
    -moz-scrollbar-width: none;
    -ms-scrollbar-width: none;
    scrollbar-width: none;
    -ms-overflow-style: none;
    -ms-scroll-snap-type: x mandatory;
    -webkit-scroll-snap-type: x mandatory;
    -moz-scroll-snap-type: x mandatory;
    -ms-scroll-snap-type: x mandatory;
    scroll-snap-type: x mandatory;
  }

  .c9::-webkit-scrollbar {
    display: none;
  }
}

@media (min-width:868px) {
  .c9 {
    display: grid;
    grid-template-columns: repeat(3,1fr);
  }
}

@media (min-width:868px) {
  .c20 {
    max-width: 670px;
    display: grid;
    grid-template-columns: repeat(3,1fr);
    gap: 10px;
    margin: 0 auto;
  }
}

@media (max-width:868px) {
  .c24 {
    margin-top: 2em;
  }
}

@media (min-width:868px) {
  .c25 {
    margin-top: 1em;
  }
}

@media (max-width:868px) {
  .c25 {
    margin-top: 5em;
  }
}

@media (max-width:868px) {

}

@media (min-width:868px) {

}

@media (min-width:868px) {
  .c15 {
    padding: 3rem;
  }
}

@media (min-width:868px) {
  .c16 {
    height: 2rem;
  }
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <header
      class="c2"
    >
      <nav
        class="c3"
      >
        <a
          class="c4"
          href="/"
        >
          <div
            class="c5"
          >
            <svg
              class="c6"
              fill="none"
              height="71"
              viewBox="0 0 52 71"
              width="52"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                logo-glif
              </title>
              <path
                d="M10.4352 26.1836C10.4354 26.2474 10.4481 26.3105 10.4727 26.3693C10.4973 26.4281 10.5332 26.4815 10.5785 26.5264C10.6237 26.5714 10.6773 26.6069 10.7363 26.6312C10.7952 26.6554 10.8584 26.6677 10.9222 26.6675H50.5292C50.593 26.6677 50.6561 26.6554 50.7151 26.6312C50.7741 26.6069 50.8277 26.5714 50.8729 26.5264C50.9181 26.4815 50.9541 26.4281 50.9786 26.3693C51.0032 26.3105 51.016 26.2474 51.0162 26.1836V10.919C51.016 10.8552 51.0032 10.7922 50.9786 10.7333C50.9541 10.6745 50.9181 10.6211 50.8729 10.5762C50.8277 10.5313 50.7741 10.4957 50.7151 10.4715C50.6561 10.4473 50.593 10.4349 50.5292 10.4351H21.3573C21.2935 10.4349 21.2304 10.4473 21.1714 10.4715C21.1124 10.4957 21.0588 10.5313 21.0136 10.5762C20.9683 10.6211 20.9324 10.6745 20.9078 10.7333C20.8833 10.7922 20.8705 10.8552 20.8703 10.919V15.7485C20.8705 15.8123 20.8833 15.8753 20.9078 15.9342C20.9324 15.993 20.9683 16.0464 21.0136 16.0913C21.0588 16.1362 21.1124 16.1718 21.1714 16.196C21.2304 16.2202 21.2935 16.2326 21.3573 16.2324H42.9C43.515 16.2324 44.1048 16.4767 44.5397 16.9116C44.9746 17.3465 45.2189 17.9363 45.2189 18.5513C45.2189 19.1663 44.9746 19.7562 44.5397 20.191C44.1048 20.6259 43.515 20.8702 42.9 20.8702H18.5514C17.9364 20.8702 17.3465 20.6259 16.9117 20.191C16.4768 19.7562 16.2325 19.1663 16.2325 18.5513V8.1162C16.2325 7.50119 16.4768 6.91136 16.9117 6.47648C17.3465 6.0416 17.9364 5.79729 18.5514 5.79729H50.5292C50.593 5.79749 50.6561 5.78513 50.7151 5.76093C50.7741 5.73672 50.8277 5.70113 50.8729 5.6562C50.9181 5.61127 50.9541 5.55787 50.9786 5.49905C51.0032 5.44024 51.016 5.37715 51.0162 5.31341V0.483881C51.0156 0.355201 50.964 0.232012 50.8727 0.141311C50.7814 0.0506101 50.6579 -0.000205419 50.5292 6.2415e-07H10.9222C10.7935 -0.000205419 10.67 0.0506101 10.5787 0.141311C10.4874 0.232012 10.4358 0.355201 10.4352 0.483881V26.1836Z"
                fill="black"
              />
              <path
                d="M50.5284 47.6879C50.6572 47.6883 50.7809 47.6376 50.8723 47.5468C50.9638 47.4561 51.0155 47.3328 51.0161 47.204V42.3706C51.0155 42.2418 50.9638 42.1185 50.8723 42.0278C50.7809 41.937 50.6572 41.8863 50.5284 41.8867H0.487747C0.358933 41.8863 0.235219 41.937 0.143772 42.0278C0.0523238 42.1185 0.000616192 42.2418 2.46584e-06 42.3706V47.2001C-0.000201704 47.2642 0.0122749 47.3278 0.0367141 47.387C0.0611532 47.4463 0.0970723 47.5001 0.142404 47.5455C0.187736 47.5908 0.241586 47.6267 0.300854 47.6512C0.360122 47.6756 0.423639 47.6881 0.487747 47.6879H50.5284Z"
                fill="black"
              />
              <path
                d="M50.5284 37.2413C50.5921 37.2416 50.6553 37.2293 50.7143 37.2052C50.7733 37.1811 50.827 37.1456 50.8723 37.1008C50.9176 37.0559 50.9536 37.0025 50.9783 36.9438C51.0029 36.885 51.0158 36.8219 51.0161 36.7582V31.9279C51.0155 31.7991 50.9638 31.6758 50.8723 31.585C50.7809 31.4943 50.6572 31.4436 50.5284 31.444H8.1162C7.50119 31.444 6.91136 31.1997 6.47648 30.7648C6.0416 30.3299 5.79729 29.7401 5.79729 29.1251V0.626417C5.7977 0.497602 5.74697 0.373889 5.65625 0.282441C5.56553 0.190993 5.44222 0.139286 5.31341 0.138672H0.483882C0.355068 0.139286 0.231763 0.190993 0.14104 0.282441C0.0503165 0.373889 -0.000408571 0.497602 2.47885e-06 0.626417V36.7582C3.22392e-05 36.8811 0.0473625 36.9993 0.13218 37.0882L0.143002 37.0998C0.187479 37.1443 0.240271 37.1795 0.298363 37.2035C0.356455 37.2275 0.418707 37.2398 0.481564 37.2397H50.5284V37.2413Z"
                fill="black"
              />
              <path
                d="M3.01713e-05 70.3858C-0.000684024 70.4499 0.0112877 70.5136 0.0352565 70.573C0.0592253 70.6325 0.0947178 70.6866 0.139691 70.7323C0.184664 70.778 0.23823 70.8143 0.297306 70.8393C0.356381 70.8642 0.419799 70.8771 0.48391 70.8774H5.31344C5.44225 70.8768 5.56556 70.8251 5.65628 70.7337C5.747 70.6422 5.79773 70.5185 5.79732 70.3897C5.79732 69.904 5.99024 69.4383 6.33365 69.0949C6.67706 68.7515 7.14283 68.5585 7.62848 68.5585H50.5284C50.6572 68.5589 50.7809 68.5082 50.8724 68.4175C50.9638 68.3268 51.0155 68.2035 51.0161 68.0746V63.2413C51.0155 63.1124 50.9638 62.9891 50.8724 62.8984C50.7809 62.8077 50.6572 62.757 50.5284 62.7574H8.11623C7.50121 62.7574 6.91139 62.5131 6.47651 62.0782C6.04163 61.6433 5.79732 61.0535 5.79732 60.4385C5.79732 59.8235 6.04163 59.2336 6.47651 58.7987C6.91139 58.3639 7.50121 58.1196 8.11623 58.1196H50.5284C50.6572 58.12 50.7809 58.0692 50.8724 57.9785C50.9638 57.8878 51.0155 57.7645 51.0161 57.6357V52.8061C51.0155 52.6773 50.9638 52.554 50.8724 52.4633C50.7809 52.3726 50.6572 52.3219 50.5284 52.3223H0.487775C0.35896 52.3219 0.235247 52.3726 0.143799 52.4633C0.0523515 52.554 0.000643898 52.6773 3.01713e-05 52.8061V70.3858Z"
                fill="black"
              />
            </svg>
          </div>
        </a>
      </nav>
      <nav
        class="c3 c7"
      >
        <a
          class="c4 c8"
          href="https://glif.io/blog"
          rel="noreferrer noopener"
          target="_blank"
        >
          Blog
        </a>
        <a
          class="c4 c8"
          href="https://github.com/glifio"
          rel="noreferrer noopener"
          target="_blank"
        >
          Code
        </a>
        <a
          class="c4 c8"
          href="https://lotus.filecoin.io/docs/developers/hosted-lotus"
          rel="noreferrer noopener"
          target="_blank"
        >
          Nodes
        </a>
      </nav>
    </header>
    <div
      class="c9"
    >
      <a
        class="c10"
        data-testid="app-tile-wrapper"
        rel="noreferrer"
        target="_blank"
      >
        <div
          class="c11"
        >
          <header
            class="c12"
          >
            <div>
              <h2
                style="font-size: 26px; margin: 0px; opacity: 1; text-decoration: none;"
              >
                Wallet
              </h2>
              
            </div>
          </header>
          <img
            alt=""
            class="c13 default"
            src="/bg-sender.jpg"
          />
          <img
            alt=""
            class="c13 hover"
            src="/bg-sender-hover.jpg"
            title="Wallet"
          />
        </div>
        <div
          class="c14"
        >
          A lightweight interface for sending and receiving Filecoin.
        </div>
      </a>
      <a
        class="c10"
        data-testid="app-tile-wrapper"
        rel="noreferrer"
        target="_blank"
      >
        <div
          class="c11"
        >
          <header
            class="c12"
          >
            <div>
              <h2
                style="font-size: 26px; margin: 0px; opacity: 1; text-decoration: none;"
              >
                Safe
              </h2>
              <h3
                style="font-size: 26px; opacity: 0.3; text-decoration: line-through; line-height: 1; margin: 0px;"
              >
                Vault
              </h3>
            </div>
          </header>
          <img
            alt=""
            class="c13 default"
            src="/bg-safe.jpg"
          />
          <img
            alt=""
            class="c13 hover"
            src="/bg-safe-hover.jpg"
            title="Safe"
          />
        </div>
        <div
          class="c14"
        >
          A Filecoin multisig.
        </div>
      </a>
      <a
        class="c10"
        data-testid="app-tile-wrapper"
        rel="noreferrer"
        target="_blank"
      >
        <div
          class="c11"
        >
          <header
            class="c12"
          >
            <div>
              <h2
                style="font-size: 26px; margin: 0px; opacity: 1; text-decoration: none;"
              >
                Explorer
              </h2>
              
              <h3
                style="font-size: 26px; opacity: 0.5; line-height: 1; margin: 0px;"
              >
                Beta
              </h3>
            </div>
          </header>
          <img
            alt=""
            class="c13 default"
            src="/bg-explorer.jpg"
          />
          <img
            alt=""
            class="c13 hover"
            src="/bg-explorer-hover.jpg"
            title="Explorer"
          />
        </div>
        <div
          class="c14"
        >
          A Filecoin network explorer.
        </div>
      </a>
      <a
        class="c10"
        data-testid="app-tile-wrapper"
        rel="noreferrer"
        target="_blank"
      >
        <div
          class="c11"
        >
          <header
            class="c12"
          >
            <div>
              <h2
                style="font-size: 26px; margin: 0px; opacity: 1; text-decoration: none;"
              >
                Verify
              </h2>
              
            </div>
          </header>
          <img
            alt=""
            class="c13 default"
            src="/bg-verifier.jpg"
          />
          <img
            alt=""
            class="c13 hover"
            src="/bg-verifier-hover.jpg"
            title="Verify"
          />
        </div>
        <div
          class="c14"
        >
          A Filecoin notary service.
        </div>
      </a>
    </div>
    <div
      class="c15 c16"
    />
    <div
      class="c15 c17"
    >
      <p>
        Instead of the fortress, there are numerous fortresses now, and various people have keys.
      </p>
      <p>
        The treasures are no longer stored in the secluded cellars and tombs of the few, but guarded by the many. A new language has evolved, used to keep the treasures safe and allow those who watch over them to communicate, directly and discretely. The hieroglyphic system consists only of geometric shapes, as an homage to the interconnected data centers with their cages, racks and cabinets.
      </p>
      <p>
        All miners shall have their own, personalised Glif, as a gesture of allegiance. (soon)
      </p>
    </div>
    <div
      class="c15 c16"
    />
    <div
      class="c15 c17 primary"
    >
      <p>
        Glif offers a suite of interoperable tools for the
         
        <a
          href="https://filecoin.io/"
          rel="noreferrer noopener"
          target="_blank"
        >
          Filecoin
        </a>
         network. With these tools, people can store, permission, and trade their data in healthy and equitable ways.
      </p>
    </div>
  </div>
  <footer
    class="c18"
  >
    <svg
      class="c19"
      height="71px"
      version="1.1"
      viewBox="0 0 52 71"
      width="52px"
      xmlns="http://www.w3.org/2000/svg"
      xmlns:xlink="http://www.w3.org/1999/xlink"
    >
      <title>
        logo-glif
      </title>
      <g
        fill="none"
        fill-rule="evenodd"
        id="Page-1"
        stroke="none"
        stroke-width="1"
      >
        <g
          fill="#FFFFFF"
          fill-rule="nonzero"
          id="logo-glif"
          transform="translate(-0.000000, 0.000000)"
        >
          <path
            d="M10.4352,26.1836 C10.4352,26.2474 10.4481,26.3105 10.4727,26.3693 C10.4973,26.4281 10.5332,26.4815 10.5785,26.5264 C10.6237,26.5714 10.6773,26.6069 10.7363,26.6312 C10.7952,26.6554 10.8584,26.6675024 10.9222,26.6675024 L50.5292,26.6675024 C50.593,26.6675024 50.6561,26.6554 50.7151,26.6312 C50.7741,26.6069 50.8277,26.5714 50.8729,26.5264 C50.9181,26.4815 50.9541,26.4281 50.9786,26.3693 C51.0032,26.3105 51.0162,26.2474 51.0162,26.1836 L51.0162,10.919 C51.0162,10.8552 51.0032,10.7922 50.9786,10.7333 C50.9541,10.6745 50.9181,10.6211 50.8729,10.5762 C50.8277,10.5313 50.7741,10.4957 50.7151,10.4715 C50.6561,10.4473 50.593,10.4349 50.5292,10.4351 L21.3573,10.4351 C21.2935,10.4349 21.2304,10.4473 21.1714,10.4715 C21.1124,10.4957 21.0588,10.5313 21.0136,10.5762 C20.9683,10.6211 20.9324,10.6745 20.9078,10.7333 C20.8833,10.7922 20.8705,10.8552 20.8703,10.919 L20.8703,15.7485 C20.8705,15.8123 20.8833,15.8753 20.9078,15.9342 C20.9324,15.993 20.9683,16.0464 21.0136,16.0913 C21.0588,16.1362 21.1124,16.1718 21.1714,16.196 C21.2304,16.2202 21.2935,16.2326 21.3573,16.2324 L42.9,16.2324 C43.515,16.2324 44.1048,16.4767 44.5397,16.9116 C44.9746,17.3465 45.2189,17.9363 45.2189,18.5513 C45.2189,19.1663 44.9746,19.7562 44.5397,20.191 C44.1048,20.6259 43.515,20.8702 42.9,20.8702 L18.5514,20.8702 C17.9364,20.8702 17.3465,20.6259 16.9117,20.191 C16.4768,19.7562 16.2325,19.1663 16.2325,18.5513 L16.2325,8.1162 C16.2325,7.50119 16.4768,6.91136 16.9117,6.47648 C17.3465,6.0416 17.9364,5.79729 18.5514,5.79729 L50.5292,5.79729 C50.593,5.79749 50.6561,5.78513 50.7151,5.76093 C50.7741,5.73672 50.8277,5.70113 50.8729,5.6562 C50.9181,5.61127 50.9541,5.55787 50.9786,5.49905 C51.0032,5.44024 51.0162,5.37715 51.0162,5.31341 L51.0162,0.483881 C51.0156,0.355201 50.964,0.232012 50.8727,0.141311 C50.7814,0.0506101 50.6579,0 50.5292,0 L10.9222,0 C10.7935,0 10.67,0.0506101 10.5787,0.141311 C10.4874,0.232012 10.4358,0.355201 10.4352,0.483881 L10.4352,26.1836 Z"
            id="Path"
          />
          <path
            d="M50.5284,47.6879024 C50.6572,47.6883 50.7809,47.6376 50.8723,47.5468 C50.9638,47.4561 51.0155,47.3328 51.0161,47.204 L51.0161,42.3706 C51.0155,42.2418 50.9638,42.1185 50.8723,42.0278 C50.7809,41.937 50.6572,41.8863 50.5284,41.8866977 L0.487747,41.8866977 C0.358933,41.8863 0.235219,41.937 0.143772,42.0278 C0.0523238001,42.1185 0.000616192051,42.2418 5.25659516e-11,42.3706 L5.25659516e-11,47.2001 C5.25659516e-11,47.2642 0.0122749001,47.3278 0.0367141001,47.387 C0.0611532001,47.4463 0.0970723001,47.5001 0.142404,47.5455 C0.187736,47.5908 0.241586,47.6267 0.300854,47.6512 C0.360122,47.6756 0.423639,47.6881 0.487747,47.6879024 L50.5284,47.6879024 Z"
            id="Path"
          />
          <path
            d="M50.5284,37.2413054 C50.5921,37.2413054 50.6553,37.2293 50.7143,37.2052 C50.7733,37.1811 50.827,37.1456 50.8723,37.1008 C50.9176,37.0559 50.9536,37.0025 50.9783,36.9438 C51.0029,36.885 51.0161,36.8219 51.0161,36.7582 L51.0161,31.9279 C51.0155,31.7991 50.9638,31.6758 50.8723,31.585 C50.7809,31.4943 50.6572,31.4436 50.5284,31.444 L8.1162,31.444 C7.50119,31.444 6.91136,31.1997 6.47648,30.7648 C6.0416,30.3299 5.79729,29.7401 5.79729,29.1251 L5.79729,0.626417 C5.7977,0.497602 5.74697,0.373889 5.65625,0.282441 C5.56553,0.190993 5.44222,0.139286 5.31341,0.138672 L0.483882,0.138672 C0.355068,0.139286 0.231763,0.190993 0.14104,0.282441 C0.0503165001,0.373889 5.71134251e-11,0.497602 5.71134251e-11,0.626417 L5.71134251e-11,36.7582 C5.71134251e-11,36.8811 0.0473625001,36.9993 0.13218,37.0882 L0.143002,37.0998 C0.187479,37.1443 0.240271,37.1795 0.298363,37.2035 C0.356455,37.2275 0.418707,37.2398 0.481564,37.2397 L50.5284,37.2397 L50.5284,37.2413054 Z"
            id="Path"
          />
          <path
            d="M0,70.3858 C-0.000684023949,70.4499 0.0112877001,70.5136 0.0352565001,70.573 C0.0592253001,70.6325 0.0947178001,70.6866 0.139691,70.7323 C0.184664,70.778 0.23823,70.8143 0.297306,70.8393 C0.356381,70.8642 0.419799,70.8771 0.48391,70.8774 L5.31344,70.8774 C5.44225,70.8768 5.56556,70.8251 5.65628,70.7337 C5.747,70.6422 5.79773,70.5185 5.79732,70.3897 C5.79732,69.904 5.99024,69.4383 6.33365,69.0949 C6.67706,68.7515 7.14283,68.5585 7.62848,68.5585 L50.5284,68.5585 C50.6572,68.5589 50.7809,68.5082 50.8724,68.4175 C50.9638,68.3268 51.0155,68.2035 51.0161,68.0746 L51.0161,63.2413 C51.0155,63.1124 50.9638,62.9891 50.8724,62.8984 C50.7809,62.8077 50.6572,62.757 50.5284,62.7574 L8.11623,62.7574 C7.50121,62.7574 6.91139,62.5131 6.47651,62.0782 C6.04163,61.6433 5.79732,61.0535 5.79732,60.4385 C5.79732,59.8235 6.04163,59.2336 6.47651,58.7987 C6.91139,58.3639 7.50121,58.1196 8.11623,58.1196 L50.5284,58.1196 C50.6572,58.12 50.7809,58.0692 50.8724,57.9785 C50.9638,57.8878 51.0155,57.7645 51.0161,57.6357 L51.0161,52.8061 C51.0155,52.6773 50.9638,52.554 50.8724,52.4633 C50.7809,52.3726 50.6572,52.3219 50.5284,52.3222977 L0.487775,52.3222977 C0.35896,52.3219 0.235247,52.3726 0.143799,52.4633 C0.0523515001,52.554 0.000643898051,52.6773 0,52.8061 L0,70.3858 Z"
            id="Path"
          />
        </g>
      </g>
    </svg>
    <div
      class="c20"
    >
      <ul
        class="c21"
      >
        <li
          class="c22"
        >
          <a
            class="c23"
            href="https://discord.gg/8Hqm3qvK"
            rel="noopenner noreferrer"
            target="_blank"
          >
            Discord
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="https://blog.glif.io"
            rel="noopenner noreferrer"
            target="_blank"
          >
            Blog
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="https://github.com/glifio"
            rel="noopenner noreferrer"
            target="_blank"
          >
            Code
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="https://lotus.filecoin.io/docs/developers/hosted-lotus/"
            rel="noopenner noreferrer"
            target="_blank"
          >
            Nodes
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="mailto:squad@glif.io"
          >
            Contact
          </a>
        </li>
      </ul>
      <ul
        class="c21"
      >
        <li
          class="c22"
        >
          <a
            class="c23"
            href="undefined"
          >
            Wallet
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="undefined"
          >
            Safe
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="undefined"
          >
            Explorer
          </a>
        </li>
        <li
          class="c22"
        >
          <a
            class="c23"
            href="undefined"
          >
            Verifier
          </a>
        </li>
      </ul>
      <div
        class="c24"
      >
        GLIF
        <br />
        Freeport, Grand Bahamas
      </div>
      <div
        class="c25"
      >
        2022 Glif for life
      </div>
    </div>
  </footer>
</div>
`;
